src/ResourceManager/Dns/Commands.Dns/Records/NewAzureDnsRescordSet.cs

```diff
@@ -51,9 +51,12 @@ public class NewAzureDnsRecordSet : DnsBaseCmdlet
         [ValidateNotNullOrEmpty]
         public RecordType RecordType { get; set; }
 
-        [Alias("Tags")]
```

Comment:
this would be a breaking change for customers. I think you should keep Tags Alias here for backward compatibility.